README.rst

```diff
@@ -172,6 +172,7 @@ Africa
 * Madagascar
 * São Tomé
 * South Africa
+* Mozambique
```

Comment:
please move "Mozambique" in the list so it's going to keep the alphabetical order.